smoke/go.sum

```diff
@@ -1,17 +1,23 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Does other */go.sum need also be updated?

Reply:
I have checked other `*/go.sum`, and only found it in smoke module.